Assets/VRTK/Prefabs/SnapDropZone/VRTK_SnapDropZone.cs

```diff
@@ -186,65 +184,15 @@ public virtual void InitaliseHighlightObject(bool removeOldObject = false)
             GenerateObjects();
         }
 
-        /// <summary>
-        /// the ForceSnap method attempts to automatically attach a valid GameObject to the snap drop zone.
-        /// </summary>
-        /// <param name="objectToSnap">The GameObject to attempt to snap.</param>
-        public virtual void ForceSnap(GameObject objectToSnap)
```

Comment:
A bunch of public methods have been removed without deprecating them. This is going to break a bunch of stuff for people. The public methods should be marked as Obsolete and the the internals of the method should be changed to do the new functionality.